x-pack/plugins/cases/public/common/mock/connectors.ts

```diff
@@ -6,12 +6,14 @@
  */
 
 import type { ActionConnector, ActionTypeConnector } from '../../../common/api';
+import { basicPush } from '../../containers/mock';
+import type { CaseConnectors } from '../../containers/types';
 
 export const connectorsMock: ActionConnector[] = [
   {
     id: 'servicenow-1',
     actionTypeId: '.servicenow',
-    name: 'My Connector',
+    name: 'My SN connector',
```

Comment:
I changed the names to be sure that the tests find the correct connector in the DOM.